config/mail.php

```diff
@@ -15,7 +15,7 @@
     |
     */
 
-    'driver' => env('MAIL_DRIVER', 'smtp'),
+    'driver' => env('MAIL_DRIVER', 'mandrill'),
```

Comment:
You don't need to change these. The `smtp` string is what it defaults to in case you don't have the key in the `.env` :smiley: